addons/project/models/project_task.py

```diff
@@ -150,7 +150,7 @@ def _read_group_personal_stage_type_ids(self, stages, domain, order):
     date_end = fields.Datetime(string='Ending Date', index=True, copy=False)
     date_assign = fields.Datetime(string='Assigning Date', copy=False, readonly=True,
         help="Date on which this task was last assigned (or unassigned). Based on this, you can get statistics on the time it usually takes to assign tasks.")
-    date_deadline = fields.Datetime(string='Deadline', index=True, copy=False, tracking=True)
+    date_deadline = fields.Datetime(string='Deadline', index=True, copy=True, tracking=True)
```

Comment:
We can directly remove it in the field definition since the default value for `copy` is True. 😁 🤓 

```suggestion
    date_deadline = fields.Datetime(string='Deadline', index=True, tracking=True)
```

See https://github.com/odoo/odoo/blob/e17d3c3166a7a5f65c06ca1fa78e1228d71b86a1/odoo/fields.py#L300